packages/manager/apps/pci-kubernetes/public/translations/add/Messages_fr_FR.json

```diff
@@ -62,19 +62,18 @@
   "kube_add_plan_footer_standard": "Gratuit durant la phase de Bêta",
   "kube_add_plan_title_free": "Free",
   "kube_add_plan_description_free": "Bénéficiez de coûts optimisés pour vos ressources de calcul associés à la simplicité opérationnelle d'une solution Kubernetes managée gratuite à vie.",
-  "kube_add_plan_content_free_control": "Plan de contrôle managé, hautement disponible",
-  "kube_add_plan_content_free_high_availability": "Haute disponibilité (1-AZ)",
+  "kube_add_plan_content_free_control": "Plan de contrôle managé, hautement disponible (1AZ)",
   "kube_add_plan_content_free_SLO": "99,5 % SLO",
-  "kube_add_plan_content_free_auto_scaling": "Fonctionnalité de mise à l'échelle des pools de nœuds (Autoscaler)",
-  "kube_add_plan_content_free_ETCD": "ETCD avec une capacité allant jusqu'à 400 Mo",
+  "kube_add_plan_content_free_auto_scaling": "Fonctionnalité de mise à l'échelle automatique des pools de nœuds (autoscaling)",
+  "kube_add_plan_content_free_ETCD": "ETCD de 400 Mo",
   "kube_add_plan_content_free_version": "Support des trois versions les plus récentes de Kubernetes",
   "kube_add_plan_content_free_100": "Jusqu'à 100 nœuds par cluster",
   "kube_add_plan_title_standard": "Standard",
   "kube_add_plan_description_standard": "Bénéficiez d'un engagement de niveau de service et d'un déploiement régional (réparti sur trois zones de disponibilité), idéal pour vos charges de travail qui exigent résilience et haute disponibilité.",
   "kube_add_plan_content_standard_SLA": "99,99 % SLA",
-  "kube_add_plan_content_standard_ETCD": "ETCD dédié allant jusqu'à 8 Go",
+  "kube_add_plan_content_standard_ETCD": "ETCD dédié de 8 Go",
   "kube_add_plan_content_standard_3AZ_control_plane": "Plan de contrôle managé, hautement disponible et tolérant à la perte d'une zone de disponibilité",
-  "kube_add_plan_content_standard_scaling": "Fonctionnalité de mise à l'échelle des noeuds",
+  "kube_add_plan_content_standard_scaling": "Fonctionnalité de mise à l'échelle automatique des pools de nœuds (autoscaling)",
```

Comment:
duplicate of `kube_add_plan_content_free_auto_scaling` key, should these translations be merged?